mkdocs.yml

```diff
@@ -814,8 +814,10 @@ extra:
       url: '/projects/connect'
   # Global variables
   site_display_name: 'Ibexa Documentation'
+  product_base_name: 'Ibexa'
   product_name: 'Ibexa DXP'
-  product_name_content: 'Ibexa Headless'
+  product_name_content: 'Ibexa Content'
```

Comment:
for master branch it should remain Ibexa Headless, for 4.5 and lower it should be Ibexa Content